src/mmw/js/src/core/views.js

```diff
@@ -727,64 +728,88 @@ var MapView = Marionette.ItemView.extend({
         return new L.GeoJSON(geom, {
             style: style,
             id: geomId,
-            pointToLayer: pointToLayer
+            pointToLayer: pointToLayer,
+            onEachFeature: function(feature, layer) {
+                layer.on('mouseover', function() {
+                    layer.setStyle(dataCatalogActiveStyle);
+                    result.set('active', true);
+                });
+
+                layer.on('mouseout', function() {
+                    if (geom.type === 'Point') {
+                        // Preserve highlight of marker if popup is open.
+                        // It will get restyled when the popup is closed.
+                        if (!layer._popup._isOpen) {
```

Comment:
The feature works but there's a console error when you hover over the point in the details view now:
![screen shot 2017-09-07 at 11 34 05 am](https://user-images.githubusercontent.com/7633670/30172182-130236c0-93c2-11e7-8067-195276058761.png)

Reply:
Ah good catch. I'll look into this.

Reply:
Addressed in e2fb98d.